packages/backend/src/database/seeds/development/02_saved_queries.ts

```diff
@@ -293,4 +294,57 @@ export async function seed(knex: Knex): Promise<void> {
             updatedByUser,
         },
     );
+
+    await savedChartModel.create(
```

Comment:
Added a new chart with `chart filters` to test a few different cases on e2e tests